### DISCUSSION TITLE
Release v1.0.1 & Roadmap for next versions

### DISCUSSION THREAD

**Discussion by JunkFood02** [JunkFood02/Seal#101] [Announcements]
# Caution
This version introduces a major bug that causes app crashes, please update to v1.0.2 for the bug fix as soon as possible.
If you're experiencing errors and cannot update to the new version currently, please go to the "Download directory" settings and toggle "Custom subdirectory" on, and leave "Path template" empty for a workaround.

## What's Changed
UI Improvement: Minor changes to color scheme and texts
Feature Update: Custom output template

Add Portuguese (Brazil) translations by @EddieMattos
Add Filipino translations by @JohnCarl2234
Add Italian translations by @NicolaBortoletto
Add Dutch translations by @AxeldeWater
Add Polish translations by Auk Lokk

**Full Changelog**: https://github.com/JunkFood02/Seal/compare/v1.0.0...v1.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/JunkFood02/Seal/releases/tag/v1.0.1'>v1.0.1</a>.</em>

**Comment by JunkFood02** [comment]
## Planned features for next versions (~v1.1.0)
- Custom Template List: Store multiple templates in the app, pick one before downloading
- Cancel Download Operation: As its name, just cancel a download in one click. Check https://github.com/yausername/youtubedl-android/issues/169#issue-1327714117 for progress

## Known issue
- Same command option cannot be used multiple times  https://github.com/yausername/youtubedl-android/issues/170#issue-1327736329

**Comment by JunkFood02** [comment]
![Screenshot_20220811-073041](https://user-images.githubusercontent.com/69683722/184040469-066453a4-71a5-4ee3-8708-390360f1ace8.png)

**Reply by Marco-9456** [reply]
This looks really good 🔥🔥
